src/sql/parts/profiler/editor/profilerInput.ts

```diff
@@ -258,7 +258,7 @@ export class ProfilerInput extends EditorInput implements IProfilerSession {
 				let columnName = this._columnMapping[key];
 				if (columnName) {
 					let value = e.values[key];
-					data[columnName] = escape(value);
+					data[columnName] = value;
```

Comment:
Might be worth it to document this so we don't try to "fix" it again in the future.

Reply:
Oh and you can remove the escape import on line 21 as well

Reply:
thanks! removed the import